api/modules/files/routes/read.js

```diff
@@ -49,20 +51,20 @@ module.exports = [{
     }
   }
 }, {
-  method: 'GET',
-  path: '/projects/{project_id}/files/meta',
+  method: `GET`,
+  path: `/projects/{project_id}/files/meta`,
   config: {
     pre: [
-      prereqs.confirmRecordExists(Model, {
-        mode: 'param',
-        requestKey: 'project_id',
-        columns: ['id', 'project_id', 'path']
+      Prerequisites.confirmRecordExists(FilesModel, {
+        mode: `param`,
+        requestKey: `project_id`,
+        columns: [`id`, `project_id`, `path`]
       }),
-      prereqs.validateUser(),
-      prereqs.validateOwnership()
+      Prerequisites.validateUser(),
+      Prerequisites.validateOwnership()
     ],
-    handler: controller.getAllAsMeta.bind(controller),
-    description: 'Retrieve a collection of file objects that belong to a single project object, based on `project_id`.',
+    handler: filesController.getAllAsMeta.bind(filesController),
```

Comment:
`handler: args => filesController.getAllAsMeta(...args),` although it feels odd that this can't just be a static function and needs explicit tying to a specific fileController instance. It suggests we might have unnecessary data capturing in the fileController class.